.github/workflows/publish.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
           --tag ${NGINX_IMAGE} \
           ./nginx
           IFS=','
-          for IMAGE_TAG in {${{ inputs.image-tags }}}
+          for IMAGE_TAG in `echo ${{ inputs.image-tags }}`
```

Comment:
Is this really necessary? :thinking: 
The outer most brackets from before should be removed, and that's that.
Like; `for IMAGE_TAG in  ${{ inputs.image-tags }}; do`

Reply:
Tried that, did not work. It wouldn't split the image-tags before looping